### DISCUSSION TITLE
Auto server joiner

### DISCUSSION THREAD

**Discussion by Youniz** [Vedza/NitroSniperGo#85] [Ideas]
Hello,

I was wondering if u could add like an auto server joiner (only on the alt tokens). But add like a random delay to it so the accounts don't get disabled.

Another feature that would be great is if the sniper sniped an invalid code it saves it temporaly and when someone send the same code again it just skips it. So it doesn't snipe the same code twice or more. This would be better and would also prevent from accounts getting rate limited because sometimes people like to spam invalid codes in chat and it also spams the webhook.

**Comment by Vedza** [comment]
Yo,
That's good ideas,
About the server joiner, do you want it to join every invite link with delay ?

**Reply by Youniz** [reply]
If it can then yes. I think the more servers u are snipping in / joining giveaways it will only increase the chances of getting nitro.

**Reply by Vedza** [reply]
Don't you think people will find out you're using a self bot and will report you ?

**Reply by Youniz** [reply]
Yea true. But only do it on the alts and not on the main account. I don't think that people are gonne report a person who just joins a server. But you could just make it an option that u could toggle on and off in the config file so people can decide if the wanne do it or not. But there would need to be a pretty random delay. because i already tried to join alot of servers on my alts and after like joining 30 servers its stops the account and it asks to verify with a phone number but even then u can still snipe on them i think.

**Reply by Vedza** [reply]
I'm going to see if it's possible and then do some test for a few days before releasing this feature

**Reply by Vedza** [reply]
If someone wants to test it before release it's available here https://github.com/Vedza/NitroSniperGo/tree/invite_sniper

**Comment by Vedza** [comment]
Just added 2nd feature suggestion (duplicate code sent)

**Reply by Youniz** [reply]
Awesome!
